content-service/src/main/resources/locale/portlet/news/News_en.properties

```diff
@@ -279,3 +279,6 @@ content.alert.success.label.translation.deleted=Translation successfully deleted
 article.automatic.translation.label=Automatic translation
 article.label.translation.originalVersion=Original version
 article.label.chooseLanguage=Choose a language
+
+# content activity
+content.activity.views=Views
```

Comment:
can you use the same existing label news.details.views=views instead ?

Reply:
done!